index.bs

```diff
@@ -815,6 +815,12 @@ An attribution source is a [=struct=] with the following items:
 :: A [=boolean=].
 : <dfn>fenced</dfn>
 :: A [=boolean=].
+: <dfn>attribution scope limit</dfn>
+:: Null or a positive 32-bit integer, number of distinct [=attribution source/attribution scopes=] allowed per [=attribution source/reporting origin=] per [=attribution source/attribution destinations|attribution destination=].
+: <dfn>attribution scopes</dfn>
+:: A [=set=] of [=strings=].
```

Comment:
This field is not nullable but the field in the header validator is. Please address this discrepancy.

Reply:
`max_event_states` and `attribution_scope_limit` are per reporting origin per destination. it's not associated with the source directly specifically.

Reply:
Done.